### PR TITLE
Docs - fix broken Examples link

### PR DESCRIPTION
**Pull request by randyh62**
remove examples.md 
correct reference link

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -10,17 +10,16 @@ sudo docker build --build-arg {ARG_1_NAME}={ARG_1_VALUE} [--build-arg {ARG_2_NAM
 
 ## ARG options
 
-* Pytorch docker: 
+* Pytorch docker:
```
Comment (lpaoletti) [inline docker/README.md:13]:
```suggestion
* PyTorch docker:
```

Comment (lpaoletti) [reply docker/README.md:13]: Perhaps need to review other occurences

File: docs/how-to/overview.rst [hunk 2]
```diff
@@ -34,7 +34,7 @@ Key Components
 - Multiple framework support and portable on PyTorch, TensorFlow, and MXNet
 - Flexible graphs to help the user create custom pipelines
 - Multicore host and multi-gpu execution for the graph
-- Support for various augmentations such as fish-eye, water, gitter, non-linear blend, etc., using the AMD ROCm Performance Primitive (RPP) library
+- Support for various augmentations such as fish-eye, water, glitter, non-linear blend, etc., using the AMD ROCm Performance Primitive (RPP) library
```
Comment (lpaoletti) [inline docs/how-to/overview.rst:37]: Should the RPP be hyperlinked?

**Review by lpaoletti — COMMENTED** [review]
Might be good to update the TOC so it matches the landing page headings
image

**Comment by randyh62** [comment]
I need approval to squash and merge and cherry-pick to published docs.

**Comment by randyh62** [comment]
@kiritigowda and @rrawther this PR is to fix broken links in the documentation that return errors when the user clicks on them. This also fixes some spelling errors to avoid meltdown during linting checks, and repair a few cosmetic inconveniences while other work was being done. It is a small PR with some very worthwhile updates, but it needs someone to approve it, or provide feedback of additional effort to undertake. Thank you for any assistance you can offer.

**Pull request closed without merging**